docs/workflow/tutorial-server-calculate-logarithm-activity.mdx

```diff
@@ -75,18 +75,19 @@ namespace App1.Workflow
     {
         public static string Action { get; } = "uuid:<uuid>::CalculateLog";
 
-        public Task<IDictionary<string, object>> Execute(IDictionary<string, object> inputs, IActivityContext context)
+        public Task<IDictionary<string, object?>> Execute(IDictionary<string, object?> inputs, IActivityContext context)
         {
             // highlight-start
+            var value = Convert.ToDouble(inputs["value"]);
             double? logBase = null;
-            if (inputs.ContainsKey("base"))
+
+            if (inputs.ContainsKey("base") && inputs["base"] != null)
             {
-                logBase = inputs["base"] as double?;
+                logBase = Convert.ToDouble(inputs["base"]);
```

Comment:
Is ToDouble nullish?  Could this fail with a nullpointerexception if inputs["base"] is null?

Reply:
I check for null beforehand.